src/emqx_mgmt_cli.erl

```diff
@@ -550,9 +550,19 @@ listeners(["stop", Proto, ListenOn]) ->
             emqx_ctl:print("Failed to stop ~s listener on ~s, error:~p~n", [Proto, ListenOn, Error])
     end;
 
+listeners(["restart", Proto, ListenOn]) ->
+    case emqx_mgmt:restart_listener(Proto, ListenOn) of
+        ok ->
+            emqx_ctl:print("Restarted ~s listener on ~s successfully.~n", [Proto, ListenOn]);
+        {error, Error} ->
+            emqx_ctl:print("Failed to restart ~s listener on ~s, error:~p~n", [Proto, ListenOn, Error])
+    end;
+
```

Comment:
Do you need to add handling of the `listeners start <Proto> <Port>` command to this PR? If so, then it needs to be added here, if not, line 564 should be removed.

Reply:
My bad. I will just remove line 564.